src/components/password/password-test.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import Password, { PasswordProps } from ".";
+
+export default {
+  title: "Password/Test",
+  includeStories: ["Default"],
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
+    labelInline: {
+      control: {
+        type: "boolean",
+      },
+    },
+    labelAlign: {
+      options: ["left", "right"],
+      control: {
+        type: "select",
+      },
+    },
+    inputWidth: {
+      control: {
+        min: 0,
+        max: 100,
+        step: 1,
+        type: "range",
+      },
+    },
+    labelWidth: {
+      control: {
+        min: 0,
+        max: 100,
+        step: 1,
+        type: "range",
+      },
+    },
+    children: {
```

Comment:
**suggestion:** I think using `children` here is redundant, instead we can replace it with `value`, which will actually change the value of the input. Something like:  `value: {
      control: {
        type: "text",
      },
    },`